src/assignment/modals/ConfirmSlice.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+	Button,
+	ButtonToolbar,
+	Modal,
+	ModalBody,
+	ModalProps,
+	Spacer,
+	Toolbar,
+	ToolbarItem,
+	ToolbarRight,
+} from '@viaa/avo2-components';
+import { AssignmentBlock, AssignmentSchema_v2 } from '@viaa/avo2-types/types/assignment';
+
+import { EDIT_ASSIGNMENT_BLOCK_LABELS } from '../assignment.const';
+
+interface ConfirmSliceProps extends Pick<ModalProps, 'isOpen' | 'onClose'> {
+	assignment: AssignmentSchema_v2;
+	block?: Pick<AssignmentBlock, 'type'>;
+	onConfirm?: () => void;
+}
+
+const ConfirmSlice: FunctionComponent<ConfirmSliceProps> = ({
```

Comment:
Zou "ConfirmDeleteAssignmentBlock" niet een duidelijkere naam zijn?

Reply:
`ConfirmSlice` duidt op het desbetreffende event; https://github.com/viaacode/avo2-client/blob/feature/slice-confirm-modal/src/assignment/views/AssignmentEdit.tsx#L168-L171

Bij het gelijktrekken van collecties kunnen we deze naar shared brengen zonder het `AssignmentBlock` deel te moeten trimmen.